src/mole_C++/utils.cpp

```diff
@@ -1,8 +1,22 @@
+/**
+ * @file utils.cpp
+ * @brief Helpers for sparse operations and MATLAB analogs
+ * @date 2024/10/15
+ * 
+ * Sparse operations that repeatedly are needed, but not 
+ * necessarily part of the Armadillo library. Some other MATLAB
+ * type functions are also here, like meshgrid.
+ * 
+ */
+
+
 #include "utils.h"
 #include <cassert>
 
 #ifdef EIGEN
 #include <eigen3/Eigen/SparseLU>
+
+
```

Comment:
nit: Why double space here?  `robinbc.cpp` does not have 2 blank lines after the includes. 

Are we adhering to some format rules? I recommend looking into applying `clang-format` to the codebase.

Reply:
We are running a linter, but it is unclear if it works for branches, or just main only.
https://github.com/csrc-sdsu/mole/blob/89cc41bf105d04727ee3294daf6c3229a0f35e9e/.github/workflows/linter.yml#L58
There is a line:
VALIDATE_ALL_CODEBASE: false
Maybe this needs more updating or are we hoping that when it is merged the linter takes care of it?
Either way, the code was changed in the newest commit.
https://github.com/csrc-sdsu/mole/blob/0295c8fdbeebed3a51260cb1276d185a8270844d/src/mole_C%2B%2B/utils.h#L1-L8